### DISCUSSION TITLE
How to pass parameters to an external app (using wrapper) [question]

### DISCUSSION THREAD

**Discussion by aducom** [e107inc/e107#4795] [Q&A Plugins] [answered]
I want to integrate an external application into E107 using a wrapper, but only if the user is logged on. I need to pass some information so that the application 'knows' that the user is logged on and retrieves the user id or login name. The wrapper plugin does not provide this, but the documentation states that there is a particular parameter that you can use. 
I'm in doubt. I can modify the existing plugin and add a field to enter parameters (which will be replaced in the plugin, i.e. [%userid%] will be replaced by the currently logged-in user). I might be able to use cookies (as that will avoid the URL being shown). What would be the best approach, or is there some other option?

**Comment by Moc** [comment, marked as answer]
This is not possible using iframes, and thus it's not possible using the Wrapper plugin.
